am-ui/webcomponents/search-results-webcomponent/src/SearchResultItem.jsx

```diff
@@ -1,8 +1,52 @@
 /* eslint-disable */
+import { useTranslation } from 'react-i18next';
+import React from 'react';
+
 function SearchResultListItem(props) {
-  const {t, i18n} = useTranslation();
+  const { t, i18n } = useTranslation();
+  /* istanbul ignore next */
+  const listElements = props.results.map((result) => {
+    return (
+      <li key={result.topic_id} class="sr-ad-block">
+        <a
+          onClick={() => props.handleClick(result)}
+          class="sr-ad-link"
+          title={result.topic_values.find((value) => value.key === 'title').value}
+        >
+          <div class="sr-ad-thumb-block">
+            <img
+              src="https://media.istockphoto.com/photos/bohemian-living-room-interior-3d-render-picture-id1182454657?k=20&m=1182454657&s=612x612&w=0&h=1xEsm7BqeicA8jYk9KlerUtGsAgzzo530l5Ak1HJdnc="
+              class="sr-ad-thumb"
+              alt="2 ½ rooms apartment"
+            />
+            <p class="sr-ad-info">
+              <span class="sr-ad-type">
+                {result.topic_values.find((value) => value.key === 'title').value}
+              </span>
+              <span class="sr-ad-price">
+                {result.topic_values.find((value) => value.key === 'price').value}
+              </span>
+              <span class="sr-ad-currency"> CHF</span>&nbsp;/&nbsp;
```

Comment:
It's just blankspace so there's a space between the slashes (1000 CHF / Month instead of 1000 CHF/Month). I used the HTML-Entity since I wasn't sure if the actual blank space would work here.